renderer/views/player.js

```diff
@@ -540,3 +548,37 @@ function cssBackgroundImageDarkGradient () {
   return 'radial-gradient(circle at center, ' +
     'rgba(0,0,0,0.4) 0%, rgba(0,0,0,1) 100%)'
 }
+
+// Format the playback time of the video
+function formatPlaybackProgress (currentTime, duration) {
```

Comment:
Actually, looks like YouTube doesn't do this. Interesting.

<img width="326" alt="screen shot 2016-05-22 at 21 17 00" src="https://cloud.githubusercontent.com/assets/121766/15460161/9754b990-2062-11e6-9bd7-2570e2d65599.png">